module-4/backend/week_two.md

```diff
@@ -5,18 +5,56 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's one difference between ES5 and ES6?
+
+  - in ES6, you can declare functions with the fat arrow notation (() => {})
+  - ES6 introduces require('..') and classes
+  - ES5 requires the use of semi-colons in some instances
```

Comment:
Yep! That might be a dangerous road to take in an interview - make sure you'd be able to provide an example for that if you bring it up 😉